cobalt/BUILD.gn

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 group("gn_all") {
-  # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  testonly = true
+  deps = [
+    "//starboard($starboard_toolchain)",
+    "//starboard/nplb",
```

Comment:
are you working on building with cobalt toolchain and getting libcxx/musl etc working ? if not , would be good to specify starboard toolchain for now and add a TODO

Reply:
talked offline and decided this isn't necessary